tools/sgen/input/FluentUIStyle.yml

```diff
@@ -311,6 +311,16 @@ MSFBorderedCardNudgeTokens extends MSFCardNudgeTokens:
     outlineColor: $Colors.Stroke.neutral1
     textColor: $Colors.Brand.shade20
 
+AP_MSFCommandBarTokens:
+  backgroundColor: [ rest: $Colors.Background.neutral4, hover: "FluentUIColor(light: $Colors.Background.neutral5, dark: $Colors.Stroke.neutral2)", pressed: "FluentUIColor(light: $Colors.Background.neutralDisabled, dark: $Colors.Background.neutral5)", selected: $Colors.Background.brandRest, disabled: $Colors.Stroke.neutral1 ]
+  barBackgroundColor: $Colors.Background.neutral1
```

Comment:
nit on the naming:
Shouldn't we have **barBackgroundColor** as just "**backgroundColor**"?

Then the ones related to the buttons, we can go with the "item" prefix?
- itemFixedIconColor
- itemIconColor
- itemIconSize
- itemInterspace